src/test/resources/stats_collector/main.gobra

```diff
@@ -0,0 +1,30 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package main
+
+import "pkg1"
+import "pkg2"
+
+func main () {
+  r@ := pkg2.Rect{width: 2, height: 3}
+  c@ := pkg2.Circle{radius: 10}
+  fold r.mem()
+  fold c.mem()
+
+  test(Area(&r), Area(&c))
+}
+
+requires acc(s.mem()) && s.Size()
+ensures ret > 0
+func Area(s pkg1.Shape) (ret int) {
+  return s.Area()
+}
+
+
+requires i > 0 && j > 0
+ensures ret > 0
+trusted
+func test(i int, j int) (ret int) {
+
+}
```

Comment:
You could probably just remove the "trusted" annotation and the method body

Reply:
The goal here is to have a trusted member, to check if the stats collector writes the trusted flag correctly. The body doesn't matter to me